exchanges/bitfinex/bitfinex.go

```diff
@@ -107,6 +116,140 @@ func (b *Bitfinex) GetPlatformStatus() (int, error) {
 	return -1, fmt.Errorf("unexpected platform status value %d", response[0])
 }
 
+// GetV2MarginFunding gets borrowing rates for margin trading
+func (b *Bitfinex) GetV2MarginFunding(symbol, amount string, period int32) (MarginV2FundingData, error) {
+	var resp []interface{}
+	var response MarginV2FundingData
+	params := make(map[string]interface{})
+	params["symbol"] = symbol
+	params["period"] = period
+	params["amount"] = amount
+	err := b.SendAuthenticatedHTTPRequest2(http.MethodPost,
+		bitfinexV2MarginFunding,
+		params,
+		&resp,
+		getAccountFees)
+	if err != nil {
+		return response, err
+	}
+	if len(resp) != 2 {
+		return response, fmt.Errorf("invalid data received")
+	}
+	avgRate, ok := resp[0].(float64)
+	if !ok {
+		return response, fmt.Errorf("failed rate")
+	}
+	avgAmount, ok := resp[1].(float64)
+	if !ok {
+		return response, fmt.Errorf("failed rate")
+	}
+	response.Symbol = symbol
+	response.RateAvg = avgRate
+	response.AmountAvg = avgAmount
+	return response, nil
+}
+
+// GetV2FundingInfo gets borrowing rates for margin trading
+func (b *Bitfinex) GetV2FundingInfo(key string) (MarginV2FundingData, error) {
+	var resp []interface{}
+	var response MarginV2FundingData
+	err := b.SendAuthenticatedHTTPRequest2(http.MethodPost,
+		fmt.Sprintf(bitfinexV2FundingInfo, key),
+		nil,
+		&resp,
+		getAccountFees)
+	if err != nil {
+		return response, err
+	}
+	if len(resp) != 2 {
+		return response, fmt.Errorf("invalid data received")
+	}
+	avgRate, ok := resp[0].(float64)
+	if !ok {
+		return response, fmt.Errorf("failed rate")
+	}
+	avgAmount, ok := resp[1].(float64)
+	if !ok {
+		return response, fmt.Errorf("failed rate")
+	}
+	response.RateAvg = avgRate
+	response.AmountAvg = avgAmount
+	return response, nil
+}
+
+// GetAccountInfoV2 gets borrowing rates for margin trading
+func (b *Bitfinex) GetAccountInfoV2() (interface{}, error) {
+	var resp interface{}
+	return resp, b.SendAuthenticatedHTTPRequest2(http.MethodPost,
+		bitfinexV2AccountInfo,
+		nil,
+		&resp,
+		getAccountFees)
+}
+
+// GetV2Balances gets borrowing rates for margin trading
+func (b *Bitfinex) GetV2Balances() (interface{}, error) {
+	var resp interface{}
+	return resp, b.SendAuthenticatedHTTPRequest2(http.MethodPost,
+		bitfinexV2Balances,
```

Comment:
Similar here, would you be able to convert this to the (response type)[https://docs.bitfinex.com/reference#rest-auth-wallets], only the first 4 in each array, since those are the only ones defined.